pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -1,21 +1,26 @@
-{ buildPythonPackage, lib, fetchFromGitHub, libcint, libxc, xcfun, blas
-, numpy, scipy, h5py
+{ buildPythonPackage, python3, lib, fetchFromGitHub, libcint, libxc, xcfun, blas, cmake
+, numpy, scipy, h5py, cppe, nose, nose-exclude
 }:
 
 buildPythonPackage rec {
   pname = "pyscf";
-  version = "1.7.6.post1";
+  version = "2.0.0";
 
   src = fetchFromGitHub {
     owner = "pyscf";
     repo = pname;
-    rev = "f6c9c6654dd9609c5e467a1edd5c2c076f793acc";
-    sha256  = "0xbwkjxxysfpqz72qn6n4a0zr2h6sprbcal8j7kzymh7swjy117w";
+    rev = "v${version}";
+    sha256  = "J6t/Oq7WE5wz2aefok5c47CeiP2DskLF//JWGNzgRpw=";
   };
 
-  # Backport from the 2.0.0 alpha releases of PySCF.
-  # H5Py > 3.3 deprecates the file modes, that PySCF sets.
-  patches = [ ./h5py.patch ];
+  # setup.py calls Cmake and passes the arguments in CMAKE_CONFIGURE_ARGS to cmake.
+  nativeBuildInputs = [ cmake ];
+  dontUseCmakeConfigure = true;
+  CMAKE_CONFIGURE_ARGS = toString [
```

Comment:
Current best practice that I've seen is NOT to have environment variables in the Nix module, but export this var in ``preBuild``. I think it's so that this variable doesn't leak to parts of the build process where it's not needed.